CMakeLists.txt

```diff
@@ -22,8 +22,17 @@ project(serac LANGUAGES CXX C)
 # MPI is required in Serac.
 set(ENABLE_MPI ON CACHE BOOL "")
 
+
+if (ENABLE_CUDA)
```

Comment:
I dont think this is needed, if it is then probably only in the codevelop  build.